example/properties.yml

```diff
@@ -36,8 +38,8 @@ scheduler_properties:
     job_reschedule_interval_millisecond: 10000
     job_reschedule_maxcount: 6
     notification_reschedule_maxcount: 3
-    url: http://10.244.4.5:6104
-
+    host: ~
```

Comment:
This has not been introduced as part of this PR. Can we modify this through a new PR?